libs/parsers/src/lib/swagger/parser.ts

```diff
@@ -288,7 +288,7 @@ export class SwaggerParser {
     return map(operations, 'operationId');
   }
 
-  getOperationByOperationId(operationId: string): Operation {
+  getOperationByOperationId(operationId: string): Operation | undefined {
     const operations = this.getOperations({ unsorted: true, excludeInternalOperations: false });
     return getPropertyValue(operations, operationId);
```

Comment:
Wasn't sure if adding tests was applicable here since it's mostly a type change + code to handle that type change. If there's anywhere you'd like me to add test coverage, let me know.